### PR TITLE
Feature/update ui for master

### PR DESCRIPTION
**Pull request by W3-Mimo**
## Description

1. Upgrade app share library version for introducing Wi-Fi qr code
2. Iconic view has been changed based on feedback
3. Remove log broadcast event and use it from sdk layer
4. Introduce diagram functionalities from sdk layer
5. Mesh init api calling mechanism has been updated
6. Byte buffer protocol update to JSON format
7. Anonymous user in contact has been added
8. Introduce crash tracking process

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version: 3.5.0
* Hardware: 8.0, 8.1, 9.0
* Toolchain:
* SDK: 3.5.0

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/src/main/java/com/w3engineers/unicef/telemesh/data/helper/RmDataHelper.java [hunk 1]
```diff
@@ -617,6 +618,13 @@ private void sendToServer(List<BulletinTrackEntity> bulletinTrackEntities) {
 
     private BroadcastCommand getBroadcastCommand() {
         Payload payload = new Payload();
+
+        GeoLocation geoLocation = new GeoLocation()
+                .setLatitude("22.845272").setLongitude("89.531472");
```
Comment (W3-Anjan) [inline app/src/main/java/com/w3engineers/unicef/telemesh/data/helper/RmDataHelper.java]: Static Geo location is set due to test the broadcast related things. during piloting to cox's bazar we need to make it dynamic

File: viper/src/main/java/com/w3engineers/ext/viper/util/lib/mesh/MeshProvider.java [hunk 2]
```diff
@@ -42,13 +45,13 @@
     private String NETWORK_PREFIX = "telemesh_t1-";
     private final String SOCKET_URL = "https://multiverse.w3engineers.com/";
```
Comment (W3-Anjan) [inline viper/src/main/java/com/w3engineers/ext/viper/util/lib/mesh/MeshProvider.java]: multiverse url is static present here

Comment (W3-Mimo) [reply viper/src/main/java/com/w3engineers/ext/viper/util/lib/mesh/MeshProvider.java]: As per our new system we update this. And use it from travis environment variables.

File: app/src/main/java/com/w3engineers/unicef/telemesh/data/helper/RmDataHelper.java (hunk 1 above)
Comment (W3-Mimo) [reply app/src/main/java/com/w3engineers/unicef/telemesh/data/helper/RmDataHelper.java]: We will work for updating geo location. But now we can use it statically for connecting dashboard and getting broadcast.

**Pull request merged**